src/Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing/Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>bin\Release\Functional.CQS.AOP.CommonTestInfrastructure.MetricsCapturing.xml</DocumentationFile>
```

Comment:
Machine-specific `DocumentationFile` paths should be changed to relative paths.
This applies to both `Debug` and `Release` configs.

----

If I recall correctly, you had some warnings that were blocking you from enabling `TreatAllWarningsAsErrors` in _all_ projects in this solution. However, if there are projects where that _can_ be enabled right now (eg. newly-added projects), consider doing so for both `Debug` and `Release` configs.

Reply:
I'm not sure this is machine-specific?

Yes, you recall correctly.  The warnings are for any of the NuGet-package-generating projects (i.e. any non-test project), because Microsoft is deprecating the `licenseUrl` field in favour of a `license` field.  All I've done is fill out information in the NuGet package tab of project properties, so I'm waiting for some Visual Studio update that will fix the problem automatically.

Reply:
Oops! Sorry, I skimmed too fast, and got too used to seeing `DocumentationFile` paths that _were_ machine-specific. I thought I saw a `C:\` or `D:\` and didn't notice that the path actually started with `bin\`. 😛